containers/unit_tests/TestUnorderedMap.hpp

```diff
@@ -55,6 +65,15 @@ struct TestInsert {
     } while (rehash_on_fail && failed_count > 0u);
 
     execution_space().fence();
+
+    if (!rehash_on_fail) {
+      for (unsigned i = 0; i < map.size(); i++) {
+        auto map_idx = expected_values(i).map_idx;
+        if (map_idx != static_cast<unsigned>(~0)) {
+          ASSERT_EQ(expected_values(map_idx).v, map.value_at(map_idx));
+        }
+      }
+    }
```

Comment:
This is running on the host but the VIew is allocated in the default memory space so our GPU CI fails:
```
 Kokkos::View ERROR: attempt to access inaccessible memory space (label="ExpectedValues")
```

Reply:
I have been marking feedback I've addressed as resolved. Is the process for the reviewee to mark comments as resolved or should the reviewer do that?